msal/application.py

```diff
@@ -238,6 +239,10 @@ class ClientApplication(object):
         "You can enable broker by following these instructions. "
         "https://msal-python.readthedocs.io/en/latest/#publicclientapplication")
 
+    _TOKEN_CACHE_DATA: dict[str, str] = {  # field_in_data: field_in_cache
```

Comment:
The type annotation dict[str, str] for _TOKEN_CACHE_DATA uses Python 3.9+ syntax. If you intend to support Python 3.7 or 3.8 (as implied by the import on line 9), consider using Dict[str, str] from typing for compatibility.

Reply:
Haha going well my friend. We are always looking at cutting edge stuff. Hope you and everyone else are well.